assets/src/edit-story/components/panels/videoPoster.js

```diff
@@ -30,15 +30,21 @@ import { __ } from '@wordpress/i18n';
  */
 import { Media } from '../form';
 import { SimplePanel } from './panel';
+import getCommonObjectValue from './utils/getCommonObjectValue';
 import getCommonValue from './utils/getCommonValue';
 
 function VideoPosterPanel({ selectedElements, onSetProperties }) {
-  const featuredMedia = getCommonValue(selectedElements, 'featuredMedia');
-  const poster = getCommonValue(selectedElements, 'poster');
-  const [state, setState] = useState({ featuredMedia, poster });
+  const resource = getCommonValue(selectedElements, 'resource');
+  const { posterId, poster } = getCommonObjectValue(
```

Comment:
Out of scope for this PR but I wonder if poster image field should even be displayed in case of multi-selection.